pkg/sql/sem/tree/datum.go

```diff
@@ -2074,6 +2074,26 @@ func MakeDJSON(d interface{}) (Datum, error) {
 	return &DJSON{j}, nil
 }
 
+func AsDJSON(e Expr) (*DJSON, bool) {
```

Comment:
Our CI/linter is complaining because this is an exported function without a comment.